aten/src/ATen/function_wrapper.py

```diff
@@ -131,6 +134,20 @@ def TypedDict(name, attrs, total=True):  # type: ignore
 # add non-virtual declaration to Tensor.cpp
 C10_TENSOR_METHOD_DEFINITION = CodeTemplate("""\
 
+// ${schema_string}
```

Comment:
if the only difference between this template and `UNBOXEDONLY` is the use of `schema_order_{cpp_signature,method_actuals}`, I think it'd be better to 
- keep a single template and feed in different values for `cpp_signature` and `method_actuals` at the call site
- add a comment above the single template noting the variants
Less redundancy, collapses more easily once the variation is no longer necessary, etc.

Reply:
ok I changed it